packages/api/src/external/carequality/process-subsequent-outbound-patient-discovery-resps.ts

```diff
@@ -0,0 +1,56 @@
+import { out } from "@metriport/core/util/log";
+import { capture } from "@metriport/core/util/notifications";
+import { errorToString } from "@metriport/shared/common/error";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { makeOutboundResultPoller } from "../ihe-gateway/outbound-result-poller-factory";
+import { updatePatientDiscoveryStatus } from "./command/update-patient-discovery-status";
+import { getCQData } from "./patient";
+import { getPatientOrFail } from "../../command/medical/patient/get-patient";
+
+dayjs.extend(duration);
+
+const context = "cq.patient.post-response.discover";
+const resultPoller = makeOutboundResultPoller();
+const MAX_SAFE_GWS = 100000;
```

Comment:
Could've used `Number.MAX_SAFE_INTEGER` - this way we don't need to wonder what's the safe max, but NABD.